src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +608,13 @@ public class SimpleEvents {
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		Skript.registerEvent("Advancement Done", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "advancement [(complete|finish.0)]")
```

Comment:
I think it's better if the event name reflects the pattern
```suggestion
		Skript.registerEvent("Advancement Complete", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "advancement [complete|finish]")
```

Reply:
I think the complete/finish part should be compulsory `(complete|finish)`